server/views/bookings/find.njk

```diff
@@ -0,0 +1,41 @@
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "../partials/showErrorSummary.njk" import showErrorSummary %}
+
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = applicationName + " - " + pageHeading  %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form action="{{ paths.people.find({ premisesId: premisesId }) }}" method="post">
```

Comment:
I feel like the heading of this page should be `Make a booking` with subheading of `Find a person by CRN` (or whatever)